### ISSUE TITLE
global_mobile_vit_v2_ocr Model Config

### ISSUE THREAD

**Issue by dlo747** [ankandrew/fast-plate-ocr#75]
Im trying to fine tune the global_mobile_vit_v2_ocr.onnx model by following the directions ( https://github.com/ankandrew/fast-plate-ocr/blob/master/examples/tutorial_fine_tune_plate_model.ipynb)

> `fast-plate-ocr train `
>   **--model-config-file ./cct_s_v1_global_model_config.yaml `**

>   --plate-config-file ./global_mobile_vit_v2_ocr_config.yaml `

>   --annotations ./train_annotations.csv `

>   --val-annotations ./valid_annotations.csv `

>   --augmentation-path /custom_augmentation.yaml `

>   --epochs 30 `
>   --batch-size 32 `
>   --output-dir trained_models/ `
>   --weights-path "global_mobile_vit_v2_ocr.keras" `
>   --label-smoothing 0.0 `
>   --weight-decay 0.0005 `
>   --lr 0.0005`

For the --model-config-file where do i find the Model Config file?

**Comment by ankandrew** [comment]
Hi, I suggest using the newer models (the `cct-*` ones) https://ankandrew.github.io/fast-plate-ocr/latest/inference/model_zoo/#model-zoo. They have better accuracy and are faster. The `mobile_vit_v2` is not available in a model config for that use case.

**Comment by dlo747** [comment]
Trying to fine tune the Blue Iris default OCR

**Comment by ankandrew** [comment]
> Blue Iris default OCR

I'm not aware of what that is, link to it?
